### DISCUSSION TITLE
English version for Data structures block

### DISCUSSION THREAD

**Discussion by mlx20** [y9vad9/kotlin-course#73] [Q&A] [answered]
When will be ready English version of courses?

**Comment by kotleni** [comment, marked as answer]
The course is constantly being developed and updated, so only with time will English be ready.

_P.S. You can help with course translation._

**Comment by y9vad9** [comment]
Hello! As @kotleni mentioned, for now, we are only on developing stage of some blocks and tracks, so we need some time to approve the content and finish it and after it will be avaliable.

You can still use https://course.y9vad9.com/en and translator for data structures block (yes, images won't be translated, but as workaround it's not bad).

Will do my best 💪
